tests/sentry/incidents/test_models.py

```diff
@@ -124,3 +126,97 @@ def test(self):
         assert incident.current_end_date == timezone.now()
         incident.date_closed = timezone.now() - timedelta(minutes=10)
         assert incident.current_end_date == timezone.now() - timedelta(minutes=10)
+
+
+class AlertRuleTriggerActionTargetTest(TestCase):
+    def test_user(self):
+        trigger = AlertRuleTriggerAction(
+            target_type=AlertRuleTriggerAction.TargetType.USER.value,
+            target_identifier=six.text_type(self.user.id),
+        )
+        assert trigger.target == self.user
+
+    def test_invalid_user(self):
+        trigger = AlertRuleTriggerAction(
+            target_type=AlertRuleTriggerAction.TargetType.USER.value, target_identifier="10000000"
+        )
+        assert trigger.target is None
+
+    def test_team(self):
+        trigger = AlertRuleTriggerAction(
+            target_type=AlertRuleTriggerAction.TargetType.TEAM.value,
+            target_identifier=six.text_type(self.team.id),
+        )
+        assert trigger.target == self.team
+
+    def test_invalid_team(self):
+        trigger = AlertRuleTriggerAction(
+            target_type=AlertRuleTriggerAction.TargetType.TEAM.value, target_identifier="10000000"
+        )
+        assert trigger.target is None
```

Comment:
Will the serializers prevent alert rules with no target later on?

Reply:
No, because the team could be deleted after the rule is created, resulting in no target.